ang/test/civicase/ContactIcon.spec.js

```diff
@@ -0,0 +1,41 @@
+/* eslint-env jasmine */
+
+(function (_) {
+  describe('ContactPopoverContent', function () {
+    var $controller, $rootScope, $scope, contactsDataServiceMock;
+    var expectedContactIcon = 'Individual';
+
+    beforeEach(module('civicase', function ($provide) {
+      contactsDataServiceMock = jasmine.createSpyObj('contactsDataService', ['getContactIconOf']);
+      contactsDataServiceMock.getContactIconOf.and.returnValue(expectedContactIcon);
+
+      $provide.value('ContactsDataService', contactsDataServiceMock);
+    }));
+
+    beforeEach(inject(function (_$controller_, _$rootScope_) {
+      $controller = _$controller_;
+      $rootScope = _$rootScope_;
+    }));
+
+    beforeEach(function () {
+      initController();
+    });
+
+    describe('when the directive initializes', function () {
+      it('requests the icon data for the cotnact', function () {
```

Comment:
Typo here.
`'requests the icon data for the contact',`

Reply:
Fix the typo at https://github.com/compucorp/uk.co.compucorp.civicase/pull/129/files#diff-9adc696351517616ef16cff294ac6ed7R252 too.